pages/search.vue

```diff
@@ -0,0 +1,211 @@
+<script setup lang="ts">
+import VCard from '@nypublicradio/nypr-design-system-vue3/v2/src/components/VCard.vue'
+import VByline from '@nypublicradio/nypr-design-system-vue3/v2/src/components/VByline.vue'
+import useImageUrl from '~~/composables/useImageUrl'
+
+const route = useRoute()
+
+const querySlug = route.query.q
+const query = ref(querySlug || '')
+
+let articles = ref(
+  await searchArticlePages({ q: querySlug }).then(({ data }) =>
+    normalizeSearchArticlePagesResponse(data)
+  )
+)
+const articlesToShow = ref(10)
+
+const isSearching = ref(false)
+async function getSearchResults() {
+  isSearching.value = true
+  articles.value = await searchArticlePages({ q: query.value }).then(
+    ({ data }) => normalizeSearchArticlePagesResponse(data)
+  )
+  isSearching.value = false
+}
+
+onMounted(() => {
+  const { $analytics } = useNuxtApp()
+  $analytics.sendPageView({ page_type: 'search_page' })
+  getSearchResults()
+})
+
+const newsletterSubmitEvent = (e) => {
+  //emitted newsletter submit event, @Matt, not exactly sure how to get this work like you mentioned.
+  // sendEvent('click_tracking', {
+  //   event_category: 'Click Tracking',
+  //   component: 'Footer',
+  //   event_label: 'Become a member',
+  // })
+}
+</script>
+<template>
+  <div>
+    <div class="search-page">
+      <div class="pb-4">
+        <section>
+          <div class="content">
+            <div class="grid gutter-x-xl">
+              <div class="col-1 hidden xl:block"></div>
+              <div class="col">
+                <div class="search-page-results pt-2">
+                  <span v-if="articles">
+                    <strong>{{ articles.length }}</strong> results for:
+                  </span>
+                </div>
+                <form id="search" class="mt-4 mb-2">
+                  <input
```

Comment:
consider initially focusing on the input text field

Reply:
at <768px width, we should probably make the input font size a little smaller... I would choose 14 for size and height